secboot/secboot_tpm.go

```diff
@@ -377,31 +374,44 @@ func SealKey(key EncryptionKey, params *SealKeyParams) error {
 		return err
 	}
 
-	// Provision the TPM as late as possible
-	if err := tpmProvision(tpm, params.TPMLockoutAuthFile); err != nil {
-		return err
+	if params.TPMProvision {
```

Comment:
what's the situation where this will be false? I don't see any change where this will be set to false anywhere, is this just to prepare for eventually not having a TPM in this codepath?